pkg/queue/concurrency_state.go

```diff
@@ -89,52 +88,28 @@ func ConcurrencyStateHandler(logger *zap.SugaredLogger, h http.Handler, pause, r
 	}
 }
 
-// Pause sends to an endpoint when request counts drop to zero
-func Pause(endpoint string) error {
-	action := ConcurrencyStateMessageBody{Action: "pause"}
+// ConcurrencyState Request sends to the concurrency state endpoint
+func ConcurrencyStateRequest(endpoint string, action ConcurrencyStateMessageBody) func() error {
 	bodyText, err := json.Marshal(action)
 	if err != nil {
-		return fmt.Errorf("unable to create request body: %w", err)
+		panic(err)
```

Comment:
Not 100% happy with a panic here, so if there's a better way to go I'm all ears

Reply:
I will do this, find a approach to relaunch a pod